engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -52,8 +53,8 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
       perRequest(requestContext, metadataBuilderProps, message)
   }
 
-  private def failBadRequest(exception: Exception, statusCode: StatusCode = StatusCodes.BadRequest) = respondWithMediaType(`application/json`) {
-    complete((statusCode, APIResponse.fail(exception).toJson.prettyPrint))
+  private def failBadRequest(t: Throwable, statusCode: StatusCode = StatusCodes.BadRequest) = respondWithMediaType(`application/json`) {
```

Comment:
out of curiosity was the widening because a throwable was coming in or just cuz?